pavelib/utils/test/suites/nose_suite.py

```diff
@@ -21,6 +21,15 @@ def __init__(self, *args, **kwargs):
         self.fail_fast = kwargs.get('fail_fast', False)
         self.run_under_coverage = kwargs.get('with_coverage', True)
         self.report_dir = Env.REPORT_DIR / self.root
+
+        # If set, put reports for run in "unique" directories.
+        # The main purpose of this is to ensure that the reports can be 'slurped'
+        # in the main jenkins flow job without overwriting the reports from other
+        # build steps. For local development/testing, this shouldn't be needed.
+        if os.environ.get("SHARD", None):
```

Comment:
Yea, this would return False if the env var is not _set_ and leave the directories as initially defined.  If it is set to 'None' though, that would result in "shard_None" being used. If you want it to be _set_ AND _False_, you can set it to an empty string.

```
>>> bool(None)
False
>>> bool('None')
True
>>> bool('')
False
```